portfolio/src/main/webapp/script.js

```diff
@@ -37,3 +37,24 @@ function addRandomFact(){
     const factContainer = document.getElementById('fact-container');
     factContainer.innerText = fact;
 }
+
+function getQuote() {
+  fetch('/data').then(response => response.json()).then((quote) => {
+    const statsListElement = document.getElementById('quote-container');
+    statsListElement.innerHTML = '';
+    statsListElement.appendChild(
+        createListElement('First Name: ' + quote.firstName));
```

Comment:
your code here assumes that the json returned is a map/dictionary

{
  "firstName" : "Brandon",
  "Age": 20
}

etc. Either:
  1.) Have the java convert a HashMap<String, String> to Json
  2.) In the java, convert a list of strings into Json, then here, do quote[0] instead of quote.firstName, and quote[1] instead of quote.lastName